include/real_time_tools/thread.hpp

```diff
@@ -114,6 +115,13 @@ namespace real_time_tools {
      * a non real time operation.
      */
     bool block_memory_;
+
+    /**
+     * @brief describes the optimization level of the CPU. this needs to be
+     * set to 0 to get proper realitime performance.
```

Comment:
The documentation could be a bit more compact, e.g. something like "Optimization level of the CPU. Set to 0 to get proper real-time performance."

Also typo in "realtime".

A general note on `@brief` (in this case it is okay but as I have seen huge text blocks in other places): This should actually be brief, i.e. ideally only one or two lines.  More elaborate description should go to the general text block which is separated by an empty line:
```
/**
 * @brief Brief description of 1-2 lines.
 *
 * Detailed description that can be as long as needed...
 */
```
The reason for this is that the brief description is shown in the overview when generating the Doxygen documentation, so it should not be too much text there.

Reply:
Done